test/Garnet.test/RespSetTest.cs

```diff
@@ -351,6 +351,35 @@ public void CanDoSetUnionStore(string key)
             Assert.IsTrue(expectedResult.OrderBy(t => t).SequenceEqual(strResult.OrderBy(t => t)));
         }
 
+        [Test]
+        public void CanDoSetInterStore()
+        {
+            string key = "key";
+            using var redis = ConnectionMultiplexer.Connect(TestUtils.GetConfig());
+            var db = redis.GetDatabase(0);
+
+            var key1 = "key1";
+            var key1Value = new RedisValue[] { "a", "b", "c" };
+
+            var key2 = "key2";
+            var key2Value = new RedisValue[] { "c", "d", "e" };
+
+            var addResult = db.SetAdd(key1, key1Value);
+            Assert.AreEqual(3, addResult);
+            addResult = db.SetAdd(key2, key2Value);
+            Assert.AreEqual(3, addResult);
+
+            var result = (int)db.Execute("SINTERSTORE", key, key1, key2);
+            Assert.AreEqual(1, result);
+
+            var membersResult = db.SetMembers(key);
+            Assert.AreEqual(1, membersResult.Length);
+            var strResult = membersResult.Select(m => m.ToString()).ToArray();
+            var expectedResult = new[] { "c" };
+            Assert.IsTrue(expectedResult.SequenceEqual(strResult));
+        }
```

Comment:
Other good test cases to add:
- Check that if `key` exists, it is being overridden
- Check that if `key1` or `key2` don't exist, the returned set is empty
- Check that if no keys are given we get a "wrong number of arguments" error
- Check that if `key1` or `key2` don't hold a set object, we get a "WRONGTYPE" error

Reply:
Tests added without last point due to lack of WRONGTYPE error support